ground/gcs/src/plugins/coreplugin/generalsettings.ui

```diff
@@ -32,6 +32,16 @@
       <property name="verticalSpacing">
        <number>2</number>
       </property>
+      <item row="12" column="0">
```

Comment:
I haven't looked at this in QT creator yet but it looks like this one now occurs in the file twice? (Automatically connect a Tau Labs USB device).